embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -1627,11 +1632,14 @@ private Map<String, Object> normalizeProperties(@Nullable Map<String, Object> pr
         }
     }
 
-    private boolean checkSdkStarted(@NonNull String action) {
+    private boolean checkSdkStartedAndLogPublicApiUsage(@NonNull String action) {
         boolean isStarted = isStarted();
         if (!isStarted) {
             internalEmbraceLogger.logSDKNotInitialized(action);
         }
+        if (telemetryService != null) {
+            telemetryService.onPublicApiCalled(action);
+        }
```

Comment:
This brings up an interesting question: do we count a public API being called if the SDK hasn't been initialized? Whichever way we go on this it would be good to document it in the `onPublicApiCalled` function kdoc

Reply:
Not yet, but if we wanted to, I'd like it to be in a different metric. It would be easy to add to the telemetry service, and it would let us differentiate between calls. Do you think we should also add this new metric?

I'll clarify the current behavior in the function kdoc!

Reply:
We have `SdkNotInintialized` error tracking somewhere else, but yeah this seems like it would be reasonable to track in this manner so we can remove the error we log, which is too heavyhanded for we need anyway

Reply:
But we wouldn't show the developers that they are trying to use something with the SDK not initialized yet. I think we should keep doing that!

If we wanted telemetry for this, I could add it on every call of `logSDKNotInitialized`, similar to what we did on checkSdkStartedAndLogPublicApiUsage